### PR TITLE
Inline mode

### PR DESCRIPTION
**Pull request by sunshinejr**
~(This needs to wait before [danger-swift#56](https://github.com/danger/danger-swift/pull/56) is merged, though)~ merged now 😉 

#### Todo:
- [x] Add documentation
- [x] Add Changelog entry

### CODE REVIEW

**Review by ashfurrow — APPROVED** [review]
Cool, nice approach with DI on those functions 👍

**Comment by sunshinejr** [comment]
**Update:** So the Danger JS and Danger Swift inline mode was merged! Danger JS is already released and @orta is playing with Danger Swift to get all the things working for the release as well. Once Danger Swift is released, I'm gonna add documentation and we could use a final review for this one!

**Update2:** IT'S GREEN EVERYBODY. Writing docs as we speak.

**Comment by sunshinejr** [comment]
@ashfurrow okay, Danger Swift is released, I've added `Changelog` entry & documentation. Should be ready for the final review 😉 

**Edit:** or wait, seems like there is one tiny bug to fix 😄 
zrzut ekranu 2018-04-01 o 01 08 48

**Edit2:** OMG, it's working. See [Harvey#17](https://github.com/Moya/Harvey/pull/17). Basically SwiftLint always returned absolute path and because we had relative ones, we could swap it after generating Violations.

File: README.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ SwiftLint.lint()
 
 That will lint the created and modified files
 
+#### Inline mode
+
+If you want the lint result shows in diff instead of comment, you can use inline_mode option. Violations that out of the diff will show in danger's fail or warn section.
```
Comment (ashfurrow) [inline README.md:36]: Is there any Danger docs or code we could link to from here, to give more context about what inline mode is?

**Review by ashfurrow — APPROVED** [review]
Looks great! Just one small note, not a blocker. Merge when you're ready 👍

File: README.md (hunk 1 above)
Comment (sunshinejr) [reply README.md:36]: Yeah, good idea. I will add a link to the release notes of Danger JS, it's really well written 👍

**Pull request merged**